test/release/examples/benchmarks/shootout/revcomp-fast.chpl

```diff
@@ -11,7 +11,7 @@ const table = initTable("ATCGGCTAUAMKRYWWSSYRKMVBHDDHBVNN\n\n");
 proc main(args: [] string) {
   const stdin = openfd(0),
         input = stdin.reader(iokind.native, locking=false,
-                             hints=ioHintSet.direct(QIO_HINT_PARALLEL)),
+                             hints=ioHintSet.fromFlag(QIO_HINT_PARALLEL)),
```

Comment:
Should this be changed to use `ioHintSet.mmap` instead?  Or did you want to do that in a separate PR?

Reply:
Good point! I'll make that change in this PR.